### ISSUE TITLE
How can I limit possible options in Cucumber Expressions?

### ISSUE THREAD

**Issue by kedziorm** [serenity-bdd/serenity-cucumber6#15]
In the "old" regex style, I could catch one of the provided, allowed String versions. How could I do that in Cucumber Expressions? For example following step definition will allow me only to provide "red" or "Blue" String and catch which of them were provided:

```
@Given(^I like '(red|Blue)' color$)
public void performAction(String oneOftheColors) {}

```

I know that in Cucumber Expressions I could just use 

```
@Given("I like red/blue colour")
```

But I need also to know which value was provided in step / catch it.

Is it possible in Cucumber Expressions?

**Comment by hertzsprung** [comment]
I think [Customer parameter types](https://cucumber.io/docs/cucumber/cucumber-expressions/#custom-parameter-types) should provide what you need.  In any case, you're probably better off asking these kinds of questions on stackoverflow.

**Comment by kedziorm** [comment]
@hertzsprung Thank you for mentioning link. I've tried it previously, but it was not working for me. This is the only reason that I asked here this question.
In fact it was, because for some reason I needed to add apostrophes to provided types, so not:

```
@ParameterType("red|blue|yellow") 
```

but rather:

```
@ParameterType("'red'|'blue'|'yellow'") 
```

since {string} is automatically handling apostrophes, I haven't thought that they will be needed.
So it was very, very simple, but I haven't found an example of usage. The example is without apostrophes.
